File: nl/tests/fib.py
def fib(n):
    if n <= 1 :
         return 1
    else:
        return  fib(n-1) + fib(n-2)
import time
fib(30)
start = time.time() * 1000

print(fib(30))
print('cost',time.time() * 1000-start)

File: nl/tests/fib.js
println = console.log;
fib = (n) => {
    if(n <= 1) {
         return 1;
    }else{
        return  fib(n-1) + fib(n-2);
    }
};
fib(30);
start = new Date().getTime();
console.log(fib(30));
console.log((new Date().getTime())-start);